hail/python/hail/methods/statgen.py

```diff
@@ -105,12 +105,15 @@ def identity_by_descent(dataset, maf=None, bounded=True, min=None, max=None) ->
     else:
         dataset = dataset.select_rows()
     dataset = dataset.select_cols().select_globals().select_entries('GT')
-    return Table._from_java(Env.hail().methods.IBD.pyApply(
-        Env.spark_backend('ibd')._to_java_ir(require_biallelic(dataset, 'ibd')._mir),
-        joption('__maf' if maf is not None else None),
-        bounded,
-        joption(min),
-        joption(max)))
+    dataset = require_biallelic(dataset, 'ibd')
+
+    return Table(MatrixToTableApply(dataset._mir, {
+        'name': 'IBD',
+        'mafFieldName': '__maf' if maf is not None else None,
+        'bounded': bounded,
+        'min': min,
+        'max': max,
+    })).persist()
```

Comment:
Is `persist()` here correct? I'm not sure what the implications of it are

Reply:
persist() is a spark concept that roughly means: "The first time you compute this thing, serialize and store the data in memory and on local disk. The next time you would need to compute it, deserialize these data instead"

I've reviewed the previous semantics and don't see a persist either in Python or Scala, so it's probably safest to leave it off.

However, I'm quite worried about what IBD is doing, so we should add benchmarks and revisit.

Reply:
so basically just remove the persist and we're good to go

Reply:
OK 👍